cmd/openqa-mon/openqa-mon_test.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import "testing"
+
+func TestAddJobs(t *testing.T) {
+    args_t := []string{"https://openqa.suse.de", "--jobs", "5701879,5701880"}
```

Comment:
Looks good! I'd like to keep OOO as default though.

```suggestion
    args_t := []string{"https://openqa.opensuse.org", "--jobs", "5701879,5701880"}
```

Reply:
done